jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/surfaces/AccordionTests.java

```diff
@@ -3,33 +3,28 @@
 import io.github.epam.TestsInit;
 import org.testng.annotations.Test;
 
-import static io.github.com.StaticSite.accordionFrame;
-import static io.github.com.StaticSite.surfaceAccordionDisabledPage;
-import static io.github.com.StaticSite.surfaceAccordionPage;
+import static io.github.com.MaterialNavigator.openSection;
+import static io.github.com.pages.surfaces.AccordionPage.*;
+import static org.testng.Assert.assertFalse;
+import static org.testng.Assert.assertTrue;
 
 public class AccordionTests extends TestsInit {
 
     @Test
-    public void defaultAccordionTest() {
-        surfaceAccordionPage.open();
-
-        accordionFrame.accordion.is().collapsed();
-        accordionFrame.accordion.is().enabled();
-        accordionFrame.accordion.expand();
-        accordionFrame.accordion.is().contentVisible();
-
-        accordionFrame.accordion.collapse();
-        accordionFrame.accordion.is().contentInvisible();
+    public void disableAccordionTest(){
+        openSection("Accordion");
```

Comment:
Please use before method like in https://github.com/jdi-testing/jdi-light/pull/2875/files